orchagent/orchdaemon.cpp

```diff
@@ -63,6 +63,7 @@ Srv6Orch *gSrv6Orch;
 FlowCounterRouteOrch *gFlowCounterRouteOrch;
 DebugCounterOrch *gDebugCounterOrch;
 MonitorOrch *gMonitorOrch;
+CustomBfdMonitorOrch *gCustomBfdMonitorOrch;
```

Comment:
Can you please rename to `BfdMonitorOrch`?

Reply:
addressed